Docs: add a note about workspace or account-level provider usage (#3074)
databricks/terraform-provider-databricks: docs/resources/system_schema.md

-> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/release-notes/release-types.html).

> -> **Note** This resource could be used with account or workspace-level provider.

Reviewer: ```suggestion
-> **Note** This resource could be only used with workspace-level provider!
```
